packages/react-next/src/components/Link/useLink.ts

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { ComposePreparedOptions } from '@fluentui/react-compose';
+import { classNamesFunction } from '../../Utilities';
+import { ILink, ILinkProps, ILinkStyleProps, ILinkStyles } from './Link.types';
+
+const getClassNames = classNamesFunction<ILinkStyleProps, ILinkStyles>({ useStaticStyles: true });
+
+/**
+ * The useLink hook processes the Link component props and returns
+ * state, slots and slotProps for consumption by the component.
+ */
+export const useLink = (props: ILinkProps, options: ComposePreparedOptions): any => {
```

Comment:
hmm, shouldn't `any` give you a lint error?

Reply:
Apparently it is turned off in `react-next`. I don't know who did it but I plan to turn it on and add exceptions in a separate PR.